util/dvsim/Testplan.py

```diff
@@ -0,0 +1,686 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Testpoint and Testplan classes for maintaining the testplan
+"""
+
+import os
+import re
+import sys
+
+import hjson
+import mistletoe
+from tabulate import tabulate
+
+
+class TestResult:
```

Comment:
The prefix Test and Testplan below seem redundant since the scope `Testplan` uniquifies them. The "go" folks recommend to "avoid stutter" (https://blog.golang.org/package-names).

Reply:
Done.